CHANGES.rst

```diff
@@ -174,6 +174,10 @@ New Features
 
   - Enhanced text representation of ``WCS`` objects. [#3604]
 
+- The `astropy.tests.helper` module is now part of the public API (and has a
```

Comment:
Oops, yeah, I know about that in theory but seem to forget almost every time out of habit.  Will fix.